scripts/createIdamUser.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+## Usage: ./createIdamUser.sh roles email [password] [surname] [forename]
+##
+## Options:
+##    - role: Comma-separated list of roles. Roles must exist in IDAM (i.e citizen,caseworker)
+##    - email: Email address
+##    - [password]: Optional but a generated password will be output if not supplied
+##    - [surname]: Optional Last name. Default to `Test`.
+##    - [forename]: Optional First name. Default to `User`.
+##
+## Create an IDAM user with the role `citizen` or any other role provided in 'roles' option
+source ./scripts/get_secrets.sh
+export $(grep -v '^#' .env | xargs)
+
+response=$(curl --location 'https://idam-api.aat.platform.hmcts.net/o/token' \
+           --header 'Content-Type: application/x-www-form-urlencoded' \
+           --data-urlencode 'grant_type=client_credentials' \
+           --data-urlencode 'client_id=pcs-frontend' \
+           --data-urlencode 'client_secret='${PCS_FRONTEND_IDAM_SECRET}'' \
+           --data-urlencode 'scope=profile roles')
+response=${response#*:\"}
+accessToken=${response%%\"*}
```

Comment:
Use Jq, something like 

accessToken=$(echo "$response" | jq -r .access_token)
```suggestion
accessToken=${response%%\"*}
```

Reply:
Updated the script